jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/ChipGroup.java

```diff
@@ -0,0 +1,158 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.ChipGroupAssert;
+import com.epam.jdi.light.angular.elements.common.Chip;
+import com.epam.jdi.light.angular.elements.interfaces.HasOrientation;
+import com.epam.jdi.light.angular.elements.interfaces.IsGroupElement;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.interfaces.base.ICoreElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.Point;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ChipGroup extends UIBaseElement<ChipGroupAssert> implements IsGroupElement<Chip>, HasOrientation {
+
+    private final String matOptions = "mat-option";
+    public String backdropField = "#chips-autocomplete-field";
+
+    @Override
+    @JDIAction("Get list of '{name}' items")
+    public List<Chip> groupElements() {
+        core().timer().wait(() -> core().finds(".mat-mdc-chip").isDisplayed());
+        return core().finds(".mat-mdc-chip").stream()
+                .map(listItem -> new Chip().setCore(Chip.class, listItem))
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    @JDIAction("Get {0} element from '{name}'")
+    public Chip getElement(String value) {
+        return groupElements().stream()
+                .filter(element -> element.getText().equals(value))
+                .findFirst()
+                .get();
+    }
+
+    @JDIAction("Get if {0} element exist in '{name}'")
+    public boolean hasElement(String value) {
+        return groupElements().stream()
+                .anyMatch(chip -> chip.getText().equals(value));
+    }
+
+    @Override
+    @JDIAction("Get if all chips elements in '{name}' is displayed")
+    public boolean isDisplayed() {
+        return groupElements().stream()
+                .allMatch(ICoreElement::isDisplayed);
+    }
+
+    @Override
+    @JDIAction("Get if all chips elements in '{name}' is enabled")
+    public boolean isEnabled() {
+        return groupElements().stream()
+                .allMatch(ICoreElement::isEnabled);
+    }
+
+    @JDIAction("Select chip element in '{name}' by '{0}'")
+    public void selectByText(String text) {
+        getElement(text).click();
```

Comment:
Sometimes the second click on element will deselect it (if it was selected before), sometimes no, but the implementation should be equal for all